packages/thirdweb/package.json

```diff
@@ -162,6 +162,7 @@
     "@radix-ui/react-focus-scope": "1.1.0",
     "@radix-ui/react-icons": "1.3.0",
     "@radix-ui/react-tooltip": "1.1.2",
+    "@shazow/whatsabi": "^0.14.1",
```

Comment:
ugh bundle size gets a big hit... not sure if its worth adding as a hard dep for a mostly dashboard specific use case.

did you look into extracting just the bytecode fn extraction bit?

Reply:
it doesn't actually make a big difference as long as ur not using the detection (size limit doesn't fail even) - I extracted it out first but we need _most_ of the logic and the rest is correctly tree-shaken